apps/web-giddh/src/app/shared/header/components/bulk-add-dialog/bulk-add-dialog.component.html

```diff
@@ -0,0 +1,117 @@
+<ng-container
+    appTranslate
+    [file]="'accounts'"
+    (localeData)="localeData = $event"
+    (commonLocaleData)="commonLocaleData = $event"
+>
+    <div class="modal-header d-flex align-items-center activity-header">
+        <h3 mat-dialog-title>{{ localeData?.bulk_add }}</h3>
+        <button class="close" mat-dialog-close>×</button>
+    </div>
+    <mat-dialog-content class="w-100 right-dialog-height bulk-dialog">
+        <div class="p-3">
+            <form class="create-acc-form" [formGroup]="bulkAddAccountForm">
+                <ng-container formArrayName="customFields">
+                    <div class="d-flex align-items-center column-gap15 mb-3 justify-content-between">
+                        <h4 class="fw-bold w-25">{{ commonLocaleData?.app_branch_name }}</h4>
+                        <div class="d-flex align-items-center obj-gap">
```

Comment:
obj-gap check this class css code whether it is applied or not

Reply:
.obj-gap is an existing class in the style-2 file, which applies a column-gap of 5px.